File: Blind_75_LeetCode_Questions/java/Binary/Q268_Missing_Number.java
class Solution {
    public int missingNumber(int[] nums) 
    {
        int len = nums.length;
        
        for(int num = 0; num <= len; ++num)
        {
            boolean find = false;

            for(int index = 0; index < len; ++index)
            {
                if(nums[index] == num)
                {
                    find = true;
                    break;
                }
            }

            if(find == false)
                return num;
        }
        return 0;
    }
}

File: Blind_75_LeetCode_Questions/python3/Binary/Q268_Missing_Number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        nums.sort()
        n = len(nums)       # let n = 3
        for i in range(0,n): # 0 ~ 3 0, 1, 2 
            if i != nums[i]: # 0 == nums[0]
                return i
        return n
